internal/adapters/operation/operation_storage_redis.go

```diff
@@ -177,14 +177,37 @@ func (r *redisOperationStorage) ListSchedulerActiveOperations(ctx context.Contex
 		return nil, errors.NewErrUnexpected("failed to list active operations for \"%s\"", schedulerName).WithError(err)
 	}
 
-	operations = make([]*operation.Operation, len(operationsIDs))
-	for i, operationID := range operationsIDs {
-		op, err := r.GetOperation(ctx, schedulerName, operationID)
-		if err != nil {
-			return nil, err
+	pipe := r.client.Pipeline()
+
+	for _, operationID := range operationsIDs {
+		pipe.HGetAll(ctx, fmt.Sprintf("operations:%s:%s", schedulerName, operationID))
+	}
+
+	var cmders []redis.Cmder
+	metrics.RunWithMetrics(operationStorageMetricLabel, func() error {
+		cmders, err = pipe.Exec(ctx)
+		return err
+	})
+
+	if err != nil {
+		return nil, errors.NewErrUnexpected("failed execute pipe for retrieving schedulers").WithError(err)
```

Comment:
```suggestion
		return nil, errors.NewErrUnexpected("failed execute pipe for retrieving operations").WithError(err)
```

Reply:
Actually, this is a blocking change :0
Did not see it!